database/migrations/2023_01_31_145731_create_stories_table.php

```diff
@@ -18,6 +18,8 @@ public function up()
             $table->timestamps();
             $table->string('name');
             $table->text('description')->nullable();
+            $table->unsignedBigInteger('epic_id');
+            $table->foreign('epic_id')->references('id')->on('epics');
```

Comment:
Riga 21 e 22 non sono da aggiungere perché poi, giustamente, li hai messi in una migration dedicata.